felles/src/main/kotlin/no/nav/familie/kontrakter/felles/Språkkode.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.familie.kontrakter.felles
+
+enum class Språkkode {
```

Comment:
Hvis man har en felles språkkode, burde den ha med engelsk også?

Reply:
Etter hvert. De to stedene den brukes per nå, støtter ikke engelsk, så jeg ville ikke leghge det inn ennå.